<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Video</class>
 <widget class="QWidget" name="Video">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VedioDisplay</string>
  </property>
  <property name="windowIcon">
   <iconset resource="camera.qrc">
    <normaloff>:/Resources/UltimageTK_LogoOnly.png</normaloff>:/Resources/UltimageTK_LogoOnly.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>640</width>
       <height>480</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0">
       <item>
        <widget class="QPushButton" name="btnBackward">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="camera.qrc">
           <normaloff>:/Resources/backward.png</normaloff>:/Resources/backward.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="sliderProgess">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnForward">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="camera.qrc">
           <normaloff>:/Resources/forward.png</normaloff>:/Resources/forward.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="labelTime">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxSpeed">
       <item>
        <property name="text">
         <string>2.0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.25</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.75</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.5</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="camera.qrc">
         <normaloff>:/Resources/play.png</normaloff>:/Resources/play.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPause">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="camera.qrc">
         <normaloff>:/Resources/pause.png</normaloff>:/Resources/pause.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStop">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="camera.qrc">
         <normaloff>:/Resources/stop.png</normaloff>:/Resources/stop.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="camera.qrc"/>
 </resources>
 <connections/>
</ui>
